src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
 amplitude.experiment.api-key=@amplitude.experiment.api-key@
+environment=@environment@
 iteratively.analytics.environment=@iteratively.analytics.environment@
 segment.analytics.write-key=@segment.analytics.write-key@
+sentry.dsn=https://33e9484001e8411693945d7d1a8a2f64@o30291.ingest.sentry.io/5956531
```

Comment:
Should DSN key be exposed like that? Does it also mean everyone knowing the public key can submit reports to our project in Sentry?

Reply:
We could "hide" it under github secret in insert during final gradle build task, but it still be visible in final binary and could be extracted form there.
I think we have to provide some kind of public endpoint to get reports. Otherwise we probably also should hide all our company emails from public appearance to avoid be spammed.

Reply:
@ArtsiomCh analytics write key will be in the final binary too, however we do hide it :)

Reply:
@ArtsiomCh , @michelkaporin , Sentry DSN is not a secret in comparison of  Segment write key, take a look on other examples:

- https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/main/src/main/java/org/asciidoc/intellij/errorHandler/SentryErrorReporter.java#L48
- https://github.com/intellij-solidity/intellij-solidity/blob/4a9669bb22f509db54ee77ea8456d1effd2db618/src/main/kotlin/me/serce/solidity/ide/errors.kt#L27

Since we don't have different projects to send crash reports, there is no reason to make it too complicated.